package.json

```diff
@@ -24,9 +24,11 @@
   },
   "homepage": "https://github.com/zowe/zowe-cli-sample-scripts#readme",
   "devDependencies": {
+    "@types/node": "^20.14.9",
     "@typescript-eslint/eslint-plugin": "^4.29.0",
     "@typescript-eslint/parser": "^4.29.0",
     "eslint": "^7.32.0",
+    "eslint-cli": "^1.1.1",
```

Comment:
The `eslint-cli` is required (I believe) by the `npm run lint` script here. Otherwise, we require people to install it globally 😋 

> "lint": "eslint \"**/*.ts\" \"**/*.js\" --ignore-pattern \"**/.eslintrc.js\""